aws-api/src/androidTest/assets/create-comment.graphql

```diff
@@ -1,8 +1,7 @@
-mutation CommentOnEvent($eventId: ID!, $content: String!, $createdAt: String!) {
-  commentOnEvent(eventId: $eventId, content: $content, createdAt: $createdAt) {
+mutation CreateComment($eventId: ID!, $commentId:String!, $content: String!, $createdAt: String!) {
+   createComment(input: {commentId: $commentId, eventId: $eventId, content: $content, createdAt: $createdAt}) {
```

Comment:
This is usually a `CreateCommentInput!` object.   But if it's working as is, maybe I'm wrong?

```suggestion
   createComment(input: CreateCommentInput!) {
```

Reply:
Same deal as the one above.